src/utils/symbol.js

```diff
@@ -0,0 +1,9 @@
+module.exports = function(key) {
+  if (Symbol && Symbol[key]) {
```

Comment:
We should probably do `typeof Symbol !== 'undefined'` here, otherwise this will trow in case `Symbol` not defined as a global.